wandb/sdk/launch/utils.py

```diff
@@ -96,10 +97,10 @@ def construct_launch_spec(
     # override base config (if supplied) with supplied args
     launch_spec = launch_config if launch_config is not None else {}
     launch_spec["uri"] = uri
-    if entity:
-        launch_spec["entity"] = entity
-    if project:
-        launch_spec["project"] = project
+    project, entity = set_project_entity_defaults(uri, project, entity, api)
```

Comment:
wait this doesn't handle the entity/project supplied in the launch_config anymore and will override it all the time i think? need to pass the config into this fn